apps/et/et-cos/perftest-image-policy.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     hmcts.github.com/prod-automated: disabled
 spec:
   filterTags:
-    pattern: '^pr-670-[a-f0-9]+-(?P<ts>[0-9]+)'
```

Comment:
We could merge changes in PET1.1.0.7 for testing rather than doing it twice now (reverting back to 670 later) but any way approved